red-pill-contracts-py/src/scoring.py

```diff
@@ -0,0 +1,39 @@
+"""Forta agent scanning for batched transactions."""
+
+import functools
+
+import ioseeth.indicators.events as iie
+import ioseeth.metrics.evasion.morphing.logic_bomb as imeml
+import ioseeth.metrics.evasion.morphing.metamorphism as imemm
+
+import src.findings as sf
+
+# TRACES ######################################################################
+
+is_trace_factory_contract_creation = functools.lru_cache(maxsize=128)(imemm.is_trace_factory_contract_creation)
+is_trace_mutant_contract_creation = functools.lru_cache(maxsize=128)(imemm.is_trace_mutant_contract_creation)
+is_trace_red_pill_contract_creation = functools.lru_cache(maxsize=128)(imeml.is_trace_red_pill_contract_creation)
+
+def score_trace(trace: dict, **kwargs) -> dict:
+    """Estimate the probabilities that the contract performs evasion techniques."""
+    # scores each evasion technique
+    __scores = {(sf.EvasionTechnique.Metamorphism, sf.MetamorphismAlert.FactoryDeployment): 0.5, (sf.EvasionTechnique.Metamorphism, sf.MetamorphismAlert.MutantDeployment): 0.5, (sf.EvasionTechnique.LogicBomb, sf.LogicBombAlert.RedPill): 0.5}
+    # update scores
```

Comment:
is this where you configure the bot just for the red pill?

Reply:
it's still extra code, i removed it